qemu/tests/cfg/balloon_memhp.cfg

```diff
@@ -23,8 +23,14 @@
     balloon_dev_devid = balloon0
     balloon_dev_add_bus = yes
     threshold = 0.12
-    ppc64,ppc64le:
+    aarch64,ppc64,ppc64le:
         threshold = 0.15
+    # Due to known issue
+    aarch64:
+        only RHEL.9.0
```

Comment:
`9.0.0`

Reply:
We don't have `9.0.0` planned so I recommend keeping it.

Reply:
corrected